eng/templates/Azure.Management.Template/src/Azure.Management.Template.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Version>0.1.0</Version>
```

Comment:
Why version 0.1.0? I'd suggest this be `1.0.0-preview.1` as that is what I'd expect folks to start at.